x-pack/legacy/plugins/code/public/components/commits/commit.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import {
+  EuiButtonIcon,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiPanel,
+  EuiText,
+  EuiTextColor,
+  EuiCopy,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { CommitInfo } from '../../../model/commit';
+import { MAIN_ROOT } from '../routes';
+import { PathTypes } from '../../common/types';
+
+const COMMIT_ID_LENGTH = 8;
+
+interface ActionProps {
+  repoUri: string;
+  commitId: string;
+}
+
+const copyLabel = i18n.translate('xpack.code.commits.copyCommitSha', {
```

Comment:
I was unsure if there was a standard for these i18n keys, please advise.

Reply:
should be good.. Let me ask the way how Kibana team want to add i18n translation items.

Reply:
should have a suffix like label:

https://github.com/elastic/kibana/blob/0af7e5776790c4cc50bf80fae416c97bbeb74a64/packages/kbn-i18n/GUIDELINE.md

Reply:
hould have a suffix like label:

https://github.com/elastic/kibana/blob/0af7e5776790c4cc50bf80fae416c97bbeb74a64/packages/kbn-i18n/GUIDELINE.md

Reply:
eada6d8